models/face_image_quality_assessment_ediffiqa/README.md

```diff
@@ -0,0 +1,37 @@
+# eDifFIQA(T)
+
+eDifFIQA(T) is a light-weight version of the models presented in the paper [eDifFIQA: Towards Efficient Face Image Quality Assessment based on Denoising Diffusion Probabilistic Models](https://ieeexplore.ieee.org/document/10468647), it achieves state-of-the-art results in the field of face image quality assessment.
+
+Notes:
+
+- The original implementation can be found [here](https://github.com/LSIbabnikz/eDifFIQA).
+- The included model combines a pretrained MobileFaceNet backbone, with a quality regression head trained using the proceedure presented in the original paper.
+- The model predicts quality scores of aligned face samples, where a higher predicted score corresponds to a higher quality of the input sample.
```

Comment:
Could you add a description on the quality of face, e.g. how many the score it needs to reach to be a good quality face image.

Reply:
It is difficult to define an exact score that correlates well with a high quality image. Instead, I have added an analysis using two datasets, XQLFW and LFW, comparing their quality score distributions. The distributions show the differences between the quality scores of mostly high quality images from LFW compared to mixed quality images from the XQLFW dataset.